sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/ReadmeSamples.java

```diff
@@ -29,11 +29,14 @@ public void auth() {
         TokenCredential credential = new DefaultAzureCredentialBuilder()
             .build();
         // create client using DefaultAzureCredential
-        LoadTestingClientBuilder builder = new LoadTestingClientBuilder()
-            .credential(credential)
-            .endpoint("<Enter Azure Load Testing Data-Plane URL>");
-        LoadTestAdministrationClient adminClient = builder.buildLoadTestAdministrationClient();
-        LoadTestRunClient testRunClient = builder.buildLoadTestRunClient();
+        LoadTestAdministrationClientBuilder adminBuilder = new LoadTestAdministrationClientBuilder()
+                .credential(credential)
+                .endpoint("<Enter Azure Load Testing Data-Plane URL>");
+        LoadTestRunClientBuilder testRunBuilder = new LoadTestRunClientBuilder()
+                .credential(credential)
+                .endpoint("<Enter Azure Load Testing Data-Plane URL>");
+        LoadTestAdministrationClient adminClient = adminBuilder.buildClient();
+        LoadTestRunClient testRunClient = testRunBuilder.buildClient();
```

Comment:
Same as above - don't need to have a builder variable and just call `buildClient()` after `endpoint()`

Reply:
Done